### PR TITLE
feature request and questions form

### PR DESCRIPTION
**Pull request by habibayassin**
Signed-off-by: habibayassin <habibayassin@aucegypt.edu>

### CODE REVIEW

**Comment by habibayassin** [comment]
image

**Comment by habibayassin** [comment]
Questions
image

File: .github/ISSUE_TEMPLATE/questions.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Questions
+description: Ask any unanswered question 
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that your question is not covered in the [documentation](https://openroad.readthedocs.io/en/latest/) before filling the form
```
Comment (vvbandeira) [inline .github/ISSUE_TEMPLATE/questions.yml:8]: Can we also add a link to this page: https://github.com/The-OpenROAD-Project/OpenROAD/discussions

File: .github/ISSUE_TEMPLATE/questions.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - type: textarea
+    id: prompt
+    attributes:
+      label: Description
+      description: A clear and concise description of your question
```
Comment (vvbandeira) [inline .github/ISSUE_TEMPLATE/questions.yml:13]:
```suggestion
      description: A clear and concise description of your question.
```

File: .github/ISSUE_TEMPLATE/questions.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: Questions
+description: Ask any unanswered question
```
Comment (vvbandeira) [inline .github/ISSUE_TEMPLATE/questions.yml:2]:
```suggestion
description: Ask any unanswered questions.
```

File: .github/ISSUE_TEMPLATE/questions.yml (hunk 1 above)
Comment (vvbandeira) [inline .github/ISSUE_TEMPLATE/questions.yml:8]:
```suggestion
        Please make sure that your question is not covered in the [documentation](https://openroad.readthedocs.io/en/latest/) before filling out the form.
```

**Review by vvbandeira — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/questions.yml (hunk 1 above)
Comment (maliberty) [reply .github/ISSUE_TEMPLATE/questions.yml:8]: I think it is better to not have a form and on https://github.com/The-OpenROAD-Project/OpenROAD/issues/new/choose just say tell people to use the dicussion page (w/link) for questions.  The other form is fine.

Comment (habibayassin) [reply .github/ISSUE_TEMPLATE/questions.yml:8]: @maliberty where exactly do I tell people to use the discussion page?

Comment (maliberty) [reply .github/ISSUE_TEMPLATE/questions.yml:8]: In the page I linked above

**Comment by habibayassin** [comment]
Removed form
image

**Comment by maliberty** [comment]
lgtm @vvbandeira to approve

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**